nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -33,17 +33,17 @@ Versions of the host prior to 5.0 were aligned with NServiceBus core. Since vers
+The `NServiceBus.Host.exe` creates a separate *service* [Application Domain](https://docs.microsoft.com/en-us/dotnet/framework/app-domains/application-domains) to run NServiceBus and the user code. The new domain is assigned a configuration file named after the dll that contains the class implementing `IConfigureThisEndpoint`. All the configuration should be done in that file (as opposed to `NServiceBus.Host.exe.config`). In most cases that means just adding the `app.config` file to the project and letting MSBuild rename it while moving to the `bin` directory.
 
-NOTE: When the type that implements `IConfigureThisEndpoint` is not specified explicitly via a `EndpointConfigurationType` application setting key in the `NServiceBus.Host.exe.config`, the host scans all assemblies to locate this type. Scanning is done in the context of the *host* application domain, not the new *service* domain. Because of that, when [redirecting assembly versions](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/redirect-assembly-versions), the `assemblyBinding` element needs to be present in both `NServiceBus.Host.exe.config` and `app.config`. Also see [Assembly Scanning](#endpoint-configuration-assembly-scanning).
+NOTE: When the type that implements `IConfigureThisEndpoint` is not specified explicitly via an `EndpointConfigurationType` application setting key in the `NServiceBus.Host.exe.config`, the host scans all assemblies to locate this type. Scanning is done in the context of the *host* application domain, not the new *service* domain. Because of that, when [redirecting assembly versions](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/redirect-assembly-versions), the `assemblyBinding` element needs to be present in both `NServiceBus.Host.exe.config` and `app.config`. Also see [Assembly Scanning](#endpoint-configuration-assembly-scanning).
 
 ## Endpoint configuration
 
 ### Assembly scanning
 
-By default, [the assembly scanning process](/nservicebus/hosting/assembly-scanning.md) of the NServiceBus Host is the same as for a regular endpoint. At startup the host scans the runtime directory to find assemblies that contain configuration for the given endpoint, i.e. classes implementing the `IConfigureThisEndpoint` interface.
+By default, [the assembly scanning process](/nservicebus/hosting/assembly-scanning.md) of the NServiceBus Host is the same as for a regular endpoint. At startup, the host scans the runtime directory to find assemblies that contain configuration for the given endpoint, i.e. classes implementing the `IConfigureThisEndpoint` interface.
 
-The scanning process can be avoided if the class containing endpoint's configuration is explicitly specified:
+The scanning process can be avoided if the class containing the endpoint's configuration is explicitly specified:
 
 snippet: ExplicitHostConfigType
 
```

Comment:
I wanted to enhance this similarly with a link to this [article](https://docs.particular.net/samples/endpoint-configuration/) but cannot find a configuration for scanning either